src/encoded/static/components/browserfeat.js

```diff
@@ -55,12 +55,6 @@ module.exports.BrowserFeat = {
                 document.body.removeChild(scrollDiv);
                 return hiddenscroll;
             })();
-
-            // UA checks; should be retired as soon as possible
-            this.feat.uaEdge = (() => navigator.userAgent.indexOf('Edge') > 0)();
-
-            // UA checks; should be retired as soon as possible
-            this.feat.uaTrident = (() => navigator.userAgent.indexOf('Trident') > 0)();
```

Comment:
I guess we could get rid of a lot of these checks (we can guarantee we have canvas, svg, flexbox, etc.), but I’m happy to keep those for now in case there’s some weird case.